lib/ansible/release.py

```diff
@@ -20,5 +20,6 @@
 __metaclass__ = type
 
 __version__ = '2.9.27'
+__tpa_version__ = 'tpa-1.0.2'
```

Comment:
din't you say this needed to be
```suggestion
__tpa_version__ = 'tpa.1.0.2'
```

Reply:
the process will change it by itself when generating the artifacts.
I though it would make sense to keep it as a version string here and let it make its thing for the artifact.